Poker.BE/Poker.BE.Domain/Game/Pot.cs

```diff
@@ -8,11 +8,46 @@ namespace Poker.BE.Domain.Game
 {
     public class Pot
     {
+        #region Fields
+        private Pot partialPot;
+        private Pot basePot;
+        private int value;
+        private int amountToClaim;
+        private List<Player> playersClaimPot;
+        #endregion
+
         #region Properties
-        public Pot PartialPot { get; set; }
+        public Pot PartialPot { get { return this.partialPot; } set { this.partialPot = value; } }
```

Comment:
got it!
Basically I just wanted it to be organized, so you can see all the fields of a class in the "Fields" region... it just look more convenient this way